src/components/layers/LayerCard.js

```diff
@@ -9,7 +9,7 @@ import styles from './styles/LayerCard.module.css'
 import LayerToolbar from './toolbar/LayerToolbar.js'
 
 const LayerCard = ({
-    title,
+    title = '',
```

Comment:
Bjørn, should we be concerned that maps saved in this old format don't display? You can see this in 2.39 if you open these same maps. The app doesn't crash (unless you open the Edit layer dialog), but the layer doesn't get displayed.

Reply:
I think we should remove these maps from the demo db. 
https://dhis2.atlassian.net/browse/DHIS2-17795